src/BaseFirestoreRepository.spec.ts

```diff
@@ -126,7 +126,16 @@ describe('BaseFirestoreRepository', () => {
           albumsSubColl.orderByAscending('releaseDate').orderByDescending('releaseDate');
         }).toThrow();
       });
+
+      it('must succeed when orderBy* function is called more than once in the same expression with different fields', async () => {
+        const pt = await bandRepository.findById('porcupine-tree');
+        const albumsSubColl = pt.albums;
+        expect(() => {
+          albumsSubColl.orderByAscending('releaseDate').orderByDescending('name');
+        }).toBeTruthy();
```

Comment:
Can you assert that calling orderBy* methods `not.toThrow()` instead of `toBeTruthy`?

Reply:
Yeah, that makes sense!
done